fedmsg_meta_fedora_infrastructure/tests/pagure.py

```diff
@@ -782,7 +782,7 @@ class TestProjectUserAdded(Base):
     project on `pagure <https://pagure.io>`_.
     """
     expected_title = "pagure.project.user.added"
-    expected_subti = 'pingou added "ralph" to project foo'
+    expected_subti = 'pingou added "ralph" to project foo with admin access'
```

Comment:
We need to keep the current test as is, just modify its class to ``LegacyTestProjectUserAdded`` and create a new ``TestProjectUserAdded`` with the new content